src/main/java/seedu/address/ui/ContactCard.java

```diff
@@ -75,11 +75,13 @@ public ContactCard(Contact contact, int displayedIndex) {
         address.setText("Address: " + contact.getAddress().value);
         emailLabel.setText("Email: ");
         email.setText(contact.getEmail().value);
-        //email.setText("Email: " + contact.getEmail().value);
         githubUsername.setText("@"+ contact.getGitHubUsername().username);
         contact.getTechStack().stream()
                 .sorted(Comparator.comparing(techStack -> techStack.techStackName))
-                .forEach(techStack -> techStacks.getChildren().add(new Label(techStack.techStackName)));
+                .forEach(techStack -> techStacks.getChildren()
+                        .add(new Label(techStack.rating == null
+                                ? techStack.techStackName
+                                : techStack.techStackName + " | " + techStack.rating)));
```

Comment:
Not a mandatory change, but maybe you could consider using `TechStack::toString` to get a formatted string instead of exposing the internal state of `TechStack` here. `TechStack::toString` does seem to have some extra brackets around its output, but I think those can be removed because I don't see them in the other fields of `Contact`.